lib/Backend/arm64/EncoderMD.cpp

```diff
@@ -1405,7 +1405,7 @@ EncoderMD::BaseAndOffsetFromSym(IR::SymOpnd *symOpnd, RegNum *pBaseReg, int32 *p
 
         if (func->HasInlinee())
         {
-            Assert(baseReg == RegSP);
+            Assert(baseReg == RegSP || (func->HasTry() && baseReg == RegR7));
```

Comment:
>RegR7 [](start = 69, length = 5)

R0 to R7 are parameter register in ARM64

Reply:
Added TODO.